ujin/programmers/day01/12916.js

```diff
@@ -0,0 +1,8 @@
+// 문자열 내 p와 y의 개수
+function solution(s) {
+  let answer = s.toLowerCase();  
+  let answerP = answer.split('p').length;  
+  let answerY = answer.split('y').length;  
```

Comment:
값이나 형태가 변하지 않는 변수는 const 로 선언해도 무방합니다.
```suggestion
  const answer = s.toLowerCase();  
  const answerP = answer.split('p').length;  
  const answerY = answer.split('y').length;  
```